aws/warehouse.go

```diff
@@ -80,6 +88,96 @@ type AccountWarehouse struct {
 	svcUserArn *string
 	store      storage.Store
 	tmp        map[string]iam.AccessKey
+	keyGC      *processaws.KeyGC
+}
+
+func (wh *AccountWarehouse) GetServiceAccounts(ctx context.Context, project string) (<-chan *clouds.Account, error) {
+	c := make(chan *clouds.Account)
+	go func() {
+		defer close(c)
+		f := func(acct *iam.User) error {
+			a := &clouds.Account{
+				ID:          aws.StringValue(acct.UserId),
+				DisplayName: aws.StringValue(acct.UserName),
+			}
+			select {
+			case c <- a:
+			case <-ctx.Done():
+				return ctx.Err()
+			}
+			return nil
+		}
+		sess, err := createSession()
+		if err !=nil {
+			return
+		}
+		svc := iam.New(sess)
+		// FIXME: get PathPrefix from config
```

Comment:
Let's make sure to explicitly call this out for Craig. Maybe he can help us by adding a way for target adapters to have custom options.